android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/Image.kt

```diff
@@ -183,8 +182,8 @@ fun ClickableImageIcon(
                 .fillMaxSize(0.9f),
             bitmap = imageConfig.decodedBitmap!!.asImageBitmap(),
             contentDescription = null,
-            contentScale = ContentScale.Crop,
-            colorFilter = ColorFilter.tint(tint ?: imageProperties.imageConfig?.color.parseColor()),
```

Comment:
why are we removing this?

Reply:
@Lentumunai-Mark Removing the color filter might interfere with the configuration of the icon color.